testdata/BUILD

```diff
@@ -796,6 +796,53 @@ java_image(
     runtime_deps = [":java_bin_as_lib"],
 )
 
+java_binary(
```

Comment:
can you remove this target, though useful for your testing we dont need it here.